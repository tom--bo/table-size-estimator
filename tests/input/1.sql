create table `t3` (
  `id` int not null auto_increment,
  `c1` int not null,
  `c2` int not null,
  primary key (`id`),
  unique key `c2` (`c2`),
  foreign key (c2) references users (id)
);
