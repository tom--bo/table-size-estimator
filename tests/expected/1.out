Input Table Definition: (Please type ^d to end input) 
successfully ended
------
1 row max size = 28 bytes .
1 row Average size = 28 bytes .
